src/main/msp/msp.c

```diff
@@ -3002,6 +3003,34 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         }
         break;
 
+#ifdef USE_DSHOT
+    case MSP2_SEND_DSHOT_COMMAND:
+        {
+            const bool armed = ARMING_FLAG(ARMED);
+
+            if (!armed) {
+                const uint8_t commandType = sbufReadU8(src);
+                const uint8_t motorIndex = sbufReadU8(src);
+                const uint8_t commandCount = sbufReadU8(src);
+
+                if (DSHOT_CMD_TYPE_BLOCKING == commandType) {
```

Comment:
Normally in C equality checks are put as `<variable> == <constant>` - unlike JavaScript there is no risk of `<variable>` being `undefined`.

Reply:
I guess this comes from C++ :)
In case of a typo (single "=")
`<variable> = <constant>` will still compile and run.
whereas
`<constant> = <variable>` will not compile indicating there is an error.

Reply:
Ok, never considered this aspect before, but actually makes sense. TIL